app/code/Magento/Checkout/view/frontend/templates/cart/form.phtml

```diff
@@ -14,7 +14,7 @@
           method="post"
           id="form-validate"
           data-mage-init='{"Magento_Checkout/js/action/update-shopping-cart":
-              {"validationURL" : "/checkout/cart/updateItemQty"}
+              {"validationURL" : "<?= $block->getValidationUrl() ?>"}
```

Comment:
the output must be escaped (see failing static test error message: https://travis-ci.org/magento/magento2/jobs/491067360)
and the corresponding Topic in the devdocs: https://devdocs.magento.com/guides/v2.3/frontend-dev-guide/templates/template-security.html

```suggestion
              {"validationURL" : "<?= $block->escapeUrl($block->getValidationUrl()) ?>"}
```